src/view/Painting.java

```diff
@@ -0,0 +1,38 @@
+package view;
+
+import language.DrawingTask;
+import language.Painter;
+
+import java.awt.Color;
+
+/**
+ * Created by Xiaxing SHI on 21/11/15.
+ */
+public class Painting implements DrawingTask {
```

Comment:
I thing the user should write its code in only one place, even if he wants to use both methods to write what he wants to draw